src/libcore/mem/maybe_uninit.rs

```diff
@@ -553,7 +553,7 @@ impl<T> MaybeUninit<T> {
     /// x.write(Some(vec![0,1,2]));
     /// let x1 = unsafe { x.read() };
     /// let x2 = unsafe { x.read() };
-    /// // We now created two copies of the same vector, leading to a double-free when
+    /// // We now created two copies of the same vector, leading to a double-free ⚠️ when
```

Comment:
Just to be clear, this ⚠︎ is supposed to be in the middle of the sentence, right?

Reply:
I added the warning sign next to the "dangerous noun". But if you think it'd be better to move it to the end of the sentence I can do that, too.